docs/elements.md

```diff
@@ -1992,6 +1993,60 @@ The `pl-card` attributes mirror the options of [Bootstrap 4 cards](https://getbo
 
 ---
 
+### `pl-template` element
+
+Displays boilerplate HTML from templates in a reusable way.
+
+#### Sample element
+
+```html
+<pl-template file-name="outerTemplate.mustache" subdirectory="templates">
+  <pl-variable name="is-open">True</pl-variable>
+  <pl-variable
+    name="problem-statement"
+    parent-directory="question"
+    subdirectory="serverFilesQuestion"
+    file-name="statement.html"
+  ></pl-variable>
+</pl-template>
+```
+
+    QUESTION = "question_path"
+    CLIENT_FILES_QUESTION = "client_files_question_path"
+    CLIENT_FILES_COURSE = "client_files_course_path"
+    SERVER_FILES_COURSE = "server_files_course_path"
+    COURSE_EXTENSIONS = "course_extensions_path"
+
+#### Customizations
+
+| Attribute          | Type                                                                                                   | Default               | Description                                                                                                    |
+| ------------------ | ------------------------------------------------------------------------------------------------------ | --------------------- | -------------------------------------------------------------------------------------------------------------- |
+| `file-name`        | string                                                                                                 | -                     | File name of the outer template to use.                                                                        |
+| `parent-directory` | `question`, `client-files-question`, `client-files-course`, `server-files-course`, `course-extensions` | `server-files-course` | Parent directory to locate `file-name`.                                                                        |
+| `subdirectory`     | string                                                                                                 | -                     | Subdirectory within parent directory to locate `file-name`.                                                    |
```

Comment:
The only one I have possible questions about is camel casing; is there any desire to change this convention for consistency with other attributes later? If so, it's easy enough to bundle changes to those other elements in here or in another PR before this one.

Reply:
I'm not _opposed_ to changing them, but I don't think there's a need for that at the moment. I acknowledge that it's slightly idiosyncratic, but I'd argue that that's ok since it matches the actual names of the directories on disk.

Reply:
Updated in https://github.com/PrairieLearn/PrairieLearn/pull/6747/commits/a0ef247641b9e185eb39ff7003b71d181e447609